README.md

```diff
@@ -106,3 +106,12 @@ Navigate to :http://localhost:8000/
 This project now contains .travis.yml file that will help run all the tests after each push. 
 For that you will need to add the keys in the build settings. For more info check : https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings
 
+
+## MongoDB
```

Comment:
Thanks for adding this, but also worth removing all the old Trello advice that's no longer relevant!

Reply:
thanks @JackMead  !